neo/test/rawiotest/tools.py

```diff
@@ -3,24 +3,5 @@
 """
 
 import logging
-import os
-
 logger = logging.getLogger("neo.test")
 
-
-def can_use_network():
-    """
-    Return True if network access is allowed
-    """
-    try:
-        import datalad
-        HAVE_DATALAD = True
-    except:
-        HAVE_DATALAD = False
-    if not HAVE_DATALAD:
-        return False
-    if os.environ.get('NOSETESTS_NO_NETWORK', False):
-        return False
-    if os.environ.get('TRAVIS') == 'true':
-        return False
-    return True
```

Comment:
@samuelgarcia, the functional approach can actually fail because on one my computers I have datalad but I don't have git-annex correct. So I think importing datalad would still be possible, but downloading the files would not be possible. So I think an environment variable is probably safer, no?

Reply:
I got this work actually so never mind.